packages/account/src/test-utils/test-message.test.ts

```diff
@@ -0,0 +1,48 @@
+import { Address } from '@fuel-ts/address';
+import { BN } from '@fuel-ts/math';
+import { randomBytes } from 'crypto';
+import { hexlify } from 'ethers';
```

Comment:
I appreciate these are dev dependencies, but would prefer referencing internal utilities (`@fuel-ts/utils`/`@fuel-ts/crypto`)

Reply:
Leftovers of olden days, thanks for commenting on it!